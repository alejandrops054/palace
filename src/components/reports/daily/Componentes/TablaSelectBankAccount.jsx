/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react';
import Table from 'react-bootstrap/Table'

const columns = [
  { field: 'Account', headerName: 'ID', width: 70 },
  { field: 'Merchant', headerName: 'Date', width: 70 },
  { field: 'Company', headerName: 'Time', width: 70 },
  { field: 'Currency', headerName: 'Inv/CK#', width: 70 },
  { field: 'Bank', headerName: 'Inv/CK#', width: 70 }
];

const rows = [
  {Account: '1578288', headerName: 'Id', width: 70}, 
  {Merchant: '1578288', headerName: 'Date', width: 70}, 
  {Company: "PER", headerName: 'Inv', width: 70}, 
  {Currency: "US", headerName: 'Paid', width: 70},
  {Bank: "No identificado", headerName: 'Paid', width: 70}
];

export default function TablaSelectBankAccount() {
  return (
        <Table striped bordered hover>
        <thead>
            <tr>
                {columns.map((data, i)=>{ return(
                    <td key={i}>{data.field}</td>
                )})}
            </tr>
        </thead>
        <tbody>
            <tr>
                {rows.map((valor) =>{ return(
                    <td rowKey={valor.Account}>{valor.Account}{valor.Merchant}{valor.Company}{valor.Currency}{valor.Bank}</td>
                )})}
            </tr>
        </tbody>
    </Table>
  );
}