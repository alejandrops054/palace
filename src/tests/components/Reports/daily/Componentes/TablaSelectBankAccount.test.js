/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import TablaSelectBankAccount from '../../../../../components/reports/daily/Componentes/TablaSelectBankAccount'

describe('TablaSelectBankAccount component',() => {
    test('debe mostrar snapshot', () => {
        
        const wrapper = shallow(<TablaSelectBankAccount/>)
        expect(wrapper).toMatchSnapshot();
    })
    
})