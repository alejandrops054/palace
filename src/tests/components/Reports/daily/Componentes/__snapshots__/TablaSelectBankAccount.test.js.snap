// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TablaSelectBankAccount component debe mostrar snapshot 1`] = `
<ForwardRef
  bordered={true}
  hover={true}
  striped={true}
>
  <thead>
    <tr>
      <td
        key="0"
      >
        Account
      </td>
      <td
        key="1"
      >
        Merchant
      </td>
      <td
        key="2"
      >
        Company
      </td>
      <td
        key="3"
      >
        Currency
      </td>
      <td
        key="4"
      >
        Bank
      </td>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        1578288
      </td>
      <td>
        1578288
      </td>
      <td>
        PER
      </td>
      <td>
        US
      </td>
      <td>
        No identificado
      </td>
    </tr>
  </tbody>
</ForwardRef>
`;
